x-pack/plugins/apm/public/hooks/useLocalStorage.ts

```diff
@@ -26,7 +24,9 @@ export function useLocalStorage<T>(key: string, defaultValue: T) {
     }
 
     return toStore;
```

Comment:
While you are at it: do we need `toStore`? 	Can't we just return immediately instead of mutating?

```ts
    const storedItem = window.localStorage.getItem(key);
    if (storedItem !== null) {
      try {
        return JSON.parse(storedItem) as T;
      } catch (err) {
        window.localStorage.removeItem(key);
        // eslint-disable-next-line no-console
        console.log(`Unable to decode: ${key}`);
      }
    }

    return defaultValue;
```